scripts/check_filename_spaces.sh

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on spaces in file names, excluding the patterns listed below.
+#
+
+options=(
+  ! -path '*/Assets.xcassets/*'
+  ! -path '*/Target Support Files'
+  ! -path '*/Local Podspecs'
+  ! -path './Interop/Firebase Component System.md'
```

Comment:
Why not rename this file rather than grandfather it?

Reply:
Done.